### ISSUE TITLE
Fable testing: TalkBack can't read government departments

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#5180]
## Description
From Fable Testing:
"When I got to the part of the form where I'm asked to select the government department I want to fund this request, TalkBack wasn't able to see the list of departments. It's able to see the number of results, but it just doesn't see the list of choices."

## Steps to reproduce
1. Use TalkBack on an Android Mobile device
2. Go to: https://forms-staging.cdssandbox.xyz/en/id/cm7ndt2zt0000lb0bdhwwjuko
3. Type in a government department

## Details
- **Browser**: Chrome
- **Operating system**: Android
- **Environment**: Staging
- **Language**: English
- **Viewport**: Mobile
- **Output**: End-user form

## Expected behaviour
End-user should be able to navigate through the list of departments

## Additional context
User was using TalkBack assistive technology

**Comment by Abi-Nada** [comment]
Closing as a duplicate of #5179
